plugins/discovery/manager.go

```diff
@@ -63,6 +64,19 @@ func (dm *Manager) Start() {
 
 	dm.stopCh = make(chan struct{})
 
+	// init configuration file and discoverer
+	cfg := dm.runConfig.DiscoveryConfig
+	if cfg.EnableRuntimePlugins {
+		log.Info("runtime plugins enabled")
+		dm.configListener = newConfigHandler(dm.runConfig.KubeClient, dm.runConfig.DiscoveryConfig)
+		if !dm.configListener.start() {
+			log.Error("timed out waiting for configmap caches to sync")
```

Comment:
Can we continue after this "error"?

Reply:
Yes, if this error occurs, we will continue with only the static config file provided at startup.